src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -75,9 +75,9 @@
     private String image;
 
     private boolean privileged;
-    
+
```

Comment:
@jglick  - for these trailing whitespaces, do you want them to stay as well?

Reply:
No strong opinion (and I do not want to slow down a potentially important fix like this one). On the one hand we do not want trailing whitespace.¹ On the other hand it is best for PRs of substance to have minimal diffs—easier to review, less likely to have merge conflicts, easier to cherry-pick, less churn in blame view, etc. My general tendency is to err on the side of keeping diffs minimal, and if whitespace or other formatting is bothersome, to make an occasional commit/PR² which _only_ cleans this up and clearly does not affect functionality.

----

¹Best to configure an editor to strip them when pressing **Enter** or whatever—but not to touch lines that were not actually inserted or edited.

²There have been some proposals to set up mandatory code style checks in Jenkins plugins which opt in, so that after a painful baseline commit which touches basically every line of code in the repository, you can forget about the issue going forward; but I do not think this has had much momentum.

Reply:
Done and done. Changed editor to only modify whitespace on modified lines.